### ISSUE TITLE
Finetune on HMDB51

### ISSUE THREAD

**Issue by Michael-Guo** [facebook/C3D#70]
I want to finetune C3D on HMDB51. My input is videos, not frames. But I met a problem. During the training, there are many warning "No data at frame XX". Just as follow :
I0126 10:30:05.908943  7274 image_io.cpp:143] No data at frame 79
I0126 10:30:08.729575  7531 image_io.cpp:143] No data at frame 79
I0126 10:30:23.699700  8262 image_io.cpp:143] No data at frame 95
I0126 10:30:25.466506  8262 image_io.cpp:143] No data at frame 47
I0126 10:30:29.031980  8519 image_io.cpp:143] No data at frame 79
I0126 10:30:33.489359  8768 image_io.cpp:143] No data at frame 47
I0126 10:30:43.336223  9258 image_io.cpp:143] No data at frame 63
I0126 10:30:45.287762  9258 image_io.cpp:143] No data at frame 47
I0126 10:30:48.550669  9515 image_io.cpp:143] No data at frame 47
I0126 10:30:51.401243  2173 solver.cpp:237] Iteration 160, lr = 0.0001
I0126 10:30:51.401480  2173 solver.cpp:87] Iteration 160, loss = 4.84928
 
Although the training can be continued, I think it is abnormal.
Why is it? Thanks a lot!

**Comment by dutran** [comment]
There are some codecs error (from ffmpeg). If the training looks fine and your loss goes down smooth, then you should be ok.

**Comment by Michael-Guo** [comment]
@dutran  Hello dutran, My loss goes down to about 0.02 from 4.8. But when the test accuracy is very low. It is only 1%! What's the problem with me. If you have trained the C3D network on HMDB51, would please share your model and solver? Thanks a lot!
